public/components/Flyout/flyouts/alertsDashboard.js

```diff
@@ -15,25 +23,25 @@ const alertsDashboard = (payload) => {
       size: 'm',
       hideCloseButton: true,
       'data-test-subj': `alertsDashboardFlyout_${trigger_name}`,
-      type: 'push',
     },
     headerProps: { hasBorder: true },
     header: (
-      <EuiText data-test-subj={`alertsDashboardFlyout_header_${trigger_name}`}>
-        <h2>{`Alerts by ${trigger_name}`}</h2>
-      </EuiText>
+      <EuiFlexGroup justifyContent="flexStart" alignItems="center">
+        <EuiFlexItem>
+          <EuiFlexGroup alignItems="center">
```

Comment:
We should add the EuiTitle as the direct child, we don't need the flex group here since there is only one child

Reply:
done